city_scrapers/utils.py

```diff
@@ -15,3 +17,19 @@ def get_key(the_dict, location_string):
         return reduce(getitem, location_string.split('.'), the_dict) or ''
     except (KeyError, TypeError):
         return None
+
+def report_error(f):
+    @wraps(f)
+    def wrapper(*args, **kwds):
```

Comment:
This is a small thing, but the convention for keyword arguments is `**kwargs`, so we should update that here from `**kwds`